### PR TITLE
Feature/poi detail

### PR DESCRIPTION
**Pull request by jcaliksygic**
poiDetail brencha, rovno to davam do mvc brenche

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ use_frameworks!
 target 'SygicMapsKit_Example' do
     
   pod 'SygicMaps', '3.2.2'
-  pod 'SygicUIKit', :git => 'https://github.com/Sygic/sygic-ui-kit-ios.git', :branch => 'feature/mvc_switch', :commit => 'HEAD'
+  pod 'SygicUIKit', :git => 'https://github.com/Sygic/sygic-ui-kit-ios.git', :branch => 'feature/poi-detail-mvc', :commit => 'HEAD'
```
Comment (marekdovjak) [inline Example/Podfile]: to asi nechceme mergnut

File: SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift [hunk 2]
```diff
@@ -71,6 +73,24 @@ public class SYMKBrowseMapViewController: UIViewController {
         mapView.addMapMarkersCluster(mapSelectionManager.clusterLayer!)
     }
     
+    // MARK: - PoiDetail
+    
+    private func showPoiDetail(with poiDetailViewModel: SYMKPoiDetailDataSource) {
```
Comment (marekdovjak) [inline SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift]: to meno viewModel by som nahradil niecim inym

File: SygicMapsKit/Classes/PoiDetail/SYMKPoiDetailDataSource.swift [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    let topOffset: CGFloat = 68
+    
+    var place: SYPlace?
+    var coordinates: SYGeoCoordinate
+    
+    lazy var contacts: [SYUIPoiDetailCellDataSource] = {
+        var contacts = [SYUIPoiDetailCellDataSource]()
+        guard let place = place else { return contacts }
+        if let phone = place.phone {
+            contacts.append(SYUIPoiDetailCellData(title: "Phone", subtitle: phone, icon: SygicIcon.call))
```
Comment (marekdovjak) [inline SygicMapsKit/Classes/PoiDetail/SYMKPoiDetailDataSource.swift:16]: treba task na podporu lokalizacie stringov

**Review by marekdovjak — APPROVED** [review]

File: SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift (hunk 2 above)
Comment (jcaliksygic) [reply SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift]: lokalna variabla premenovana na poiDetailData typu SYMKPoiDetailDataSource

File: SygicMapsKit/Classes/PoiDetail/SYMKPoiDetailDataSource.swift (hunk 3 above)
Comment (jcaliksygic) [reply SygicMapsKit/Classes/PoiDetail/SYMKPoiDetailDataSource.swift:16]: hodil som task do backlogu... poriesi sa

**Review by Kraci — APPROVED** [review]

**Pull request merged**